inc/views/post_layout.php

```diff
@@ -207,16 +214,35 @@ public function render_cover_header() {
 		if ( $title_mode ) {
 			$title_meta_wrap_classes[] = 'nv-is-boxed';
 		}
-		$alignment = apply_filters( 'neve_post_title_alignment', '' );
-		if ( $container_mode === 'contained' ) {
-			$container_class[] = $alignment;
-		} else {
-			$main_class[] = $alignment;
+
+		/**
+		 * Filters the post title styles to override specific styles.
+		 *
+		 * @param string $style The styles for the title.
+		 * @param string $context The context of the layout (e.g. 'cover', 'normal'). Default is 'normal'.
+		 *
+		 * @since 3.1.0
+		 */
+		$cover_style = apply_filters( 'neve_post_title_alignment_style', $cover_style, 'cover' );
+		if ( ! empty( $cover_style ) ) {
+			$cover_style = 'style="' . $cover_style . '"';
+		}
+
+		/**
+		 * Keep this for legacy purpose
+		 */
+		if ( ! neve_is_new_skin() ) {
```

Comment:
@preda-bogdan let's remove this legacy part as the CSS was removed shortly after we've added this feature and the alignment never worked. This feature is supposed to work only on new skin so we don't need this.